packages/icons/src/general/IconIndicatorGuestSafetyBadge.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
@apiel51 This looks wrong? Please revert.

Reply:
@milesj Yeah this was not intentional, had some experimental changes on my fork's master branch and accidentally pulled from that instead of the base repo. Removed the files so they won't be pulled in👍